pegasus/sites.v3/code.org/views/homepage_hero.haml

```diff
@@ -47,6 +47,7 @@
   - heroes_arranged.each_with_index do |entry, index|
     -# note: this gsheet display parameter unused due to IE8 fix.
     - display = entry[:active_s] == "hide" ? "none" : "block"
+    - entry[:image_t] = entry[:image_t].gsub("/images/", "/images/fit-1500/")
```

Comment:
we load different resolutions of these images whether we're on a retina display or not.  on a non-retina display i think this is actually upscaling in some cases.  might be easiest to not do scaling for this particular image.